src/plugins/intel_cpu/src/cache/lru_cache.h

```diff
@@ -31,7 +31,7 @@ class LruCache {
      * @param value
      */
 
-    void put(Key key, Value val) {
+    void put(const Key &key, const Value &val) {
```

Comment:
Given that the values are passed by const reference now (in other words - not copied) we have to remove usage of std::move on lines 41, 46 and 47